gltf/gltf_system.rs

```diff
@@ -401,12 +404,11 @@ impl GltfModel {
     ///
     /// - Panics if the scene is out of range.
     ///
-    pub fn draw_scene(&self, scene_id: usize, viewport_dimensions: [u32; 2],
+    pub fn draw_scene(&self, scene: gltf::Scene, viewport_dimensions: [u32; 2],
```

Comment:
I'd prefer is this was still the scene ID. Passing directly a `Scene` that doesn't belong to the original document will lead to subtle bugs. But if you pass a scene ID instead, then it's binary: either it works or it panics.

Reply:
Makes sense. Will fix.